WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCouponViewModel.swift

```diff
@@ -258,6 +282,53 @@ final class AddEditCouponViewModel: ObservableObject {
         codeField = code
     }
 
+    func configureWarningBehavior() {
+        let amountSubtitleDefaultText = amountSubtitleDefaultText
+        let warningColor = Color(.warning)
```

Comment:
Totally forgot about the color adjustment! Thank you 🙇, I've defined the new semantic color by inverting the original `.warning` light and dark colors as Joe suggested. Fixed here: https://github.com/woocommerce/woocommerce-ios/pull/7036/commits/5c42287764034974a4c0d223285305507fec6c79

Let me know if this would be the expected fix.